tests/jobs/test_backfill_job.py

```diff
@@ -1538,3 +1538,25 @@ def test_mapped_dag(self, dag_id):
             executor=SequentialExecutor(),
         )
         job.run()
+
+    @pytest.mark.long_running
+    def test_mapped_dag_debug(self):
+        # Test that mapped tasks work with the DebugExecutor -- since that runs a lot more things in the same
+        # process than other executors we want to make sure users can use this to develop mapped DAGs
+        # successfully.
+        from airflow.executors.debug_executor import DebugExecutor
+
+        dag_id = "test_mapped_classic"
+        self.dagbag.process_file(str(TEST_DAGS_FOLDER / f'{dag_id}.py'))
+        dag = self.dagbag.get_dag(dag_id)
+
+        # This needs a real executor to run, so that the `make_list` task can write out the TaskMap
+
+        job = BackfillJob(
+            dag=dag,
+            start_date=days_ago(1),
+            end_date=days_ago(1),
```

Comment:
We’re deprecating `days_ago` so let’s not make the pending refactoring more work.